packages/dx-react-scheduler/docs/guides/appointment-tooltip.md

```diff
@@ -0,0 +1,31 @@
+# React Scheduler - Appointment Tooltip
+
+The React Scheduler component's `AppointmentTooltip` plugin displays compact appointment data.
+
+## Related Plugins
+
+The following plugins display an appointment data:
+
+- [AppointmentTooltip](../reference/appointment-tooltip.md) - displays an appointment information in a short form
+
+## Basic Usage
+
+Import the plugin listed above to enable a user to show an appointment information data. The appointment tooltip will be shown by a `click` on an appointment.
+
+### Uncontrolled Mode
+
+The following demo demonstrates simple way to display an appointment data.
+
+.embedded-demo({ "path": "scheduler-tooltip/uncontrolled", "showThemeSelector": true })
+
+### Controlled Mode
+
+In controlled mode, you should control the all/particular `AppointmentTooltip's` state. Pass the `visible` property and `onVisibilityChange`  event to control the visibility externally. Pass the [appointmentMeta](../reference/appointment-tooltip.md#appointmentmeta) property and `onAppointmentMetaChange` event to control the display data externally.
```

Comment:
What does all/particular mean?

Reply:
The component has 2 state properties, you can change them both and one by one.